crux-core/src/crux/io.clj

```diff
@@ -219,3 +219,9 @@
        ~@body
        (finally
          (.unlock lock# stamp#)))))
+
+(defn prn-edn [xs]
```

Comment:
LGTM, though nitpick - this function should be called `pr-edn-str` or something similar - the `n` prefix implies that there would be a newline (see `pr` vs `prn` etc.) So a function called `prn-edn` sounds like it prints edn to standard out, followed by newline.

Reply:
Alright, seems entirely reasonable! :)